app/models/event.rb

```diff
@@ -0,0 +1,27 @@
+class Event < ActiveRecord::Base
+  KINDS = %w[bucket_created bucket_moved_to_funding bucket_funded]
+
+  belongs_to :eventable, polymorphic: true
+  belongs_to :group
+  belongs_to :user
+
+  scope :sequenced, -> { where('sequence_id is not null').order('sequence_id asc') }
+  scope :chronologically, -> { order('created_at asc') }
+
+  after_create :notify_webhooks!, if: :group
+
+  validates_inclusion_of :kind, :in => KINDS
+  validates_presence_of :eventable
+
+  acts_as_sequenced scope: :group_id, column: :sequence_id, skip: lambda {|e| e.group.nil? || e.group_id.nil? }
```

Comment:
Er, we use this to say events have a sequence_id, which is a better way to order things than by ID or created_at. You shouldn't need it for webhook stuff.